### ISSUE TITLE
Curious about protos...

### ISSUE THREAD

**Issue by inkawhich** [deeplearning-wisc/sorl#1]
Hello, I have a (hopefully) simple question which stems from something being in the code that I was not necessarily expecting after reading the paper...

_What is the purpose of a SupSpectral model keeping track of prototypes during model training?_

It doesn't seem like they are used in the computation of the training loss, and it doesn't seem like they are used to evaluate "test" accuracy because the test-time centroids are zero initialized 
https://github.com/deeplearning-wisc/sorl/blob/456cac79b6c49a6302240842f1d2fca4a89f4afc/run_sorl.py#L190

Are these prototypes just for stat tracking during training and are functionally useless? Or, are they more important and I'm missing something?

Thanks in advance for the help,
Nate

**Comment by sunyiyou** [comment]
Hi Nate, 

Yes, they are some "deprecated functionalities". You can ignore them for now. 

Thanks.
